documentation.md

```diff
@@ -19,11 +19,11 @@ For previous versions of the specification, please see the [Specification Links]
 Meta-schemas
 ------------
 
-The meta-schemas are the schemas which define the JSON Schema and Hyper-Schema formats.
-
+The meta-schemas define the JSON Schema and Hyper-Schema formats using the JSON Schema format.
```

Comment:
technically the hyper-schema meta-schema is itself a hyper-schema.  I'm being super-pedantic here but if we're entertaining tiny wording tweaks...

While we're at it, "describe" would be better than "define":  the meta-schemas are informative rather than normative and it would be good to convey that better as it's been a point of confusion in the recent past.  Only the specifications define the format, and there are some aspects of the specification that either can't be expressed fully in a schema or aren't worth the complication.